.github/workflows/ci.yaml

```diff
@@ -82,21 +85,27 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: 
+        os:
           [
-            [self-hosted, macos, amd64, 13, test], 
-            [self-hosted, macos, amd64, 12, test], 
-            [self-hosted, macos, arm64, 13, test], 
+            [self-hosted, macos, amd64, 13, test],
+            [self-hosted, macos, amd64, 12, test],
+            [self-hosted, macos, arm64, 13, test],
             [self-hosted, macos, arm64, 12, test]
           ]
-    runs-on: ${{ matrix.os }} 
+    runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v3
         with:
           # We need to get all the git tags to make version injection work. See VERSION in Makefile for more detail.
           fetch-depth: 0
           persist-credentials: false
           submodules: true
+      - name: configure aws credentials
```

Comment:
I think we have to scope this to only run on push/merge, see https://github.com/aws/s2n-quic/blob/0ebd9560194364b2419aabbacf1c73634801b74d/.github/workflows/book.yml#L48-L53 for an example